apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -116,20 +122,20 @@ export default connect(
     static propTypes = {
       // from redux
       commandHistory: PropTypes.instanceOf(CommandHistory),
-      logOutput: experiments.isEnabled('react-inspector')
-        ? PropTypes.oneOfType([PropTypes.array, PropTypes.object]).isRequired
-        : PropTypes.string.isRequired,
+      logOutput: PropTypes.any.isRequired,
```

Comment:
rather than any, should this be oneOfType array, object, or string? (prevents int, bool, and func)

Reply:
Yup, I'll refactor it to `PropTypes.oneOfType([PropTypes.array, PropTypes.object, PropTypes.string])`

Reply:
I think this is just for supporting experiment flag on and off — once the experiment flag is removed and this ships to everyone, it should go back to `PropTypes.array`.